modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -101,6 +106,7 @@ template<typename Net> class Params {
               , std::tuple_size<typename Net::OutArgs>::value // num_out
               , detail::ParamDesc::Kind::Import
               , false
+              , {}
```

Comment:
false - `is_generic`
{ }     - `config`
{ }     - `reshape_in_shapes`
 
Where is initialization for `reshape_in_names` ?

Reply:
Fixed.